build/NuSpecs/ProjectReunionMetaPackage.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>$ID$</id>
+    <version>0.0.0-SpecifyVersionOnCommandline</version>
+    <title>Microsoft Project Reunion Meta Package</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <description>Project Reunion Meta Package</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <tags>Windows Reunion ProjectReunion</tags>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="file">license.txt</license>
+    <projectUrl>https://github.com/microsoft/projectreunion</projectUrl>
+    <iconUrl>https://aka.ms/winui_icon</iconUrl>
+    <dependencies> 
+      <!-- List all Project Reunion references here, please maintain in alphabetical order-->
```

Comment:
Should be enforced by tooling. Could be a PR-complete gate, could be a build task or nightly task or... Not saying tooling should automagically sort it; possibly, or could possibly merely monitor and flag incorrectness for human follow up

Please create an issue to track the need for tooling to verify correctness